storops/unity/resource/lun.py

```diff
@@ -388,11 +388,17 @@ def _do_check_move_session(move_session_id):
                 return False
 
         clz = storops.unity.resource.move_session.UnityMoveSession
-        is_compressed = self.is_data_reduction_enabled
-
+        is_compressed = True if provision == 'compressed' else \
+            self.is_data_reduction_enabled
+        is_thin = True
```

Comment:
Could use `is_thin = None` when `provision = None` to keep default behavior unchange.